torchvision/transforms/functional.py

```diff
@@ -424,22 +424,24 @@ def pad(img: Tensor, padding: List[int], fill: int = 0, padding_mode: str = "con
             This value is only used when the padding_mode is constant.
             Only number is supported for torch Tensor.
             Only int or str or tuple value is supported for PIL Image.
-        padding_mode: Type of padding. Should be: constant, edge, reflect or symmetric. Default is constant.
+        padding_mode (str): Type of padding. Should be: constant, edge, reflect or symmetric.
+            Default is constant.
 
             - constant: pads with a constant value, this value is specified with fill
 
-            - edge: pads with the last value on the edge of the image,
-                    if input a 5D torch Tensor, the last 3 dimensions will be padded instead of the last 2
+            - edge: pads with the last value at the edge of the image
```

Comment:
With the propsoed changes the docs look like this:


![image](https://user-images.githubusercontent.com/1190450/115530093-5ed36c00-a28b-11eb-96c3-50a864e36a0a.png)

----- 


Perhaps it would look slightly better like this:


![image](https://user-images.githubusercontent.com/1190450/115530053-54b16d80-a28b-11eb-9d1f-24c3d3278737.png)


Which  requires the next line starting on the same column of the `e` of `edge`, i.e.:

```
            - edge: pads with the last value at the edge of the image.
              If input a 5D torch Tensor, the last 3 dimensions will be padded instead of the last 2

            - reflect: pads with reflection of image without repeating the last value on the edge.
              For example, padding [1, 2, 3, 4] with 2 elements on both sides in reflect mode
              will result in [3, 2, 1, 2, 3, 4, 3, 2]

            - symmetric: pads with reflection of image repeating the last value on the edge.
              For example, padding [1, 2, 3, 4] with 2 elements on both sides in symmetric mode
              will result in [2, 1, 1, 2, 3, 4, 4, 3]
```

(You'll need to add full dots at the end of the sentences otherwise it will not read well).

WDYT?

Reply:
@NicolasHug Yes I think your suggestion does look better (tighter but smoother), I'll commit again in a bit.

Reply:
Changed as requested 87c4885